backend/app/kafka/consumer_vid.py

```
@@ -0,0 +1,144 @@
+
+            try:
+                response = s3_client_obj.delete_from_s3(
+                    s3_BUCKET_UNTRANSCODED,
+                    video_file
+                )
+
+                if response:
+                    print(f"Object {video_file} deleted successfully.\n\n")
+                    print(response)
+                else:
+                    raise Exception("Delete failed")
```

Ruff (0.15.15)

[warning] Abstract `raise` to an inner function

(TRY301)